COMPILING-FLATPAK.md

```diff
@@ -0,0 +1,34 @@
+# COMPILING GUIDE FOR LINUX (USING FLATPAK)
```

Comment:
I think sentence-cased headings will look better.

Reply:
You mean like this?

# Compiling Guide for Linux